linux_os/guide/system/software/integrity/crypto/ssh_client_rekey_limit/tests/bad_size.fail.sh

```diff
@@ -0,0 +1,4 @@
+# platform = multi_platform_all
+
+sed -e '/RekeyLimit/d' /etc/ssh/sshd_config
```

Comment:
This is a wrong file, this configures the deamon, not the client, it should be `/etc/ssh/ssh_config` or according to your rule.yml it should be `/etc/ssh/ssh_config.d/02-rekey-limit.conf. This problem is in multiple other test scenarios.